corehq/apps/userreports/laboratory/adapter.py

```diff
@@ -0,0 +1,41 @@
+from corehq.apps.userreports.adapter import IndicatorAdapter
+from corehq.apps.userreports.es.adapter import IndicatorESAdapter
+from corehq.apps.userreports.sql.adapter import IndicatorSqlAdapter
+
+
+class IndicatorLaboratoryAdapter(IndicatorAdapter):
+
+    def __init__(self, config):
+        super(IndicatorLaboratoryAdapter, self).__init__(config)
+        self.es_adapter = IndicatorESAdapter(config)
+        self.sql_adapter = IndicatorSqlAdapter(config)
+
+    def rebuild_table(self):
+        self.es_adapter.rebuild_table()
+        self.sql_adapter.rebuild_table()
+
+    def drop_table(self):
+        self.es_adapter.drop_table()
+        self.sql_adapter.drop_table()
+
+    def rebuild_table_if_necessary(self):
+        self.es_adapter.rebuild_table_if_necessary()
+        # sql doesn't have this
```

Comment:
This is the result of a little hackiness right now. Added a todo in the rebuild code to move that to an adapter